### ISSUE TITLE
Google api key

### ISSUE THREAD

**Issue by oakwoodgates** [oakwoodgates/happyhour#30]
https://console.developers.google.com

**Comment by BenhamL** [comment]
do you still need this from me?

**Comment by oakwoodgates** [comment]
@BenhamL yes, we're using mine so it's not a priority. I'll probably have to help you set it up, it's kinda complicated, but once it's done you don't have to worry about it again.
